### PR TITLE
Remove region syncer and region storage

### PR DESCRIPTION
**Pull request by YangKeao**
This PR contains #39 (to ease the trouble to merge), please merge it first.

### CODE REVIEW

File: scheduler/server/core/storage.go [hunk 1]
```diff
@@ -16,17 +16,15 @@ package core
...
-	"strconv"
-	"strings"
-	"sync/atomic"
-
 	"github.com/gogo/protobuf/proto"
 	"github.com/pingcap-incubator/tinykv/proto/pkg/metapb"
 	"github.com/pingcap-incubator/tinykv/scheduler/server/kv"
 	"github.com/pkg/errors"
 	"go.etcd.io/etcd/clientv3"
+	"math"
```
Comment (Connor1996) [inline scheduler/server/core/storage.go]: please keep the order as before

Comment (YangKeao) [reply scheduler/server/core/storage.go]: Oops, this is a mistake of my IDE :cry:. Should we add `goimports -w -l ./` into Makefile to automatically group imports?

Comment (Connor1996) [reply scheduler/server/core/storage.go]: you can add it in format check

Comment (YangKeao) [reply scheduler/server/core/storage.go]: OK! I will do it in another PR.

**Pull request merged**